static/app/views/aiAnalytics/tokenUsageChart.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled';
+
+import type {PageFilters} from 'sentry/types';
+import {MetricDisplayType} from 'sentry/utils/metrics/types';
+import {useMetricsQuery} from 'sentry/utils/metrics/useMetricsQuery';
+import withPageFilters from 'sentry/utils/withPageFilters';
+import {MetricChartContainer} from 'sentry/views/dashboards/metrics/chart';
+
+interface TokenUsageChartProps {
+  metric: 'ai.total_tokens.used' | 'ai.prompt_tokens.used' | 'ai.completion_tokens.used';
+  selection: PageFilters;
+  isGlobalSelectionReady?: boolean;
+}
+
+function TokenUsageChart({selection, metric}: TokenUsageChartProps) {
+  const {
+    data: timeseriesData,
+    isLoading,
+    isError,
+    error,
+  } = useMetricsQuery(
+    [
+      {
+        name: 'a',
+        mri: `c:custom/${metric}@tokens`,
```

Comment:
This is fine for the poc, but i don't think we should keep using the custom namespace for these metrics.